terraform/cloudformation/prod/main.tf

```diff
@@ -0,0 +1,4 @@
+module "demo" {
+  source = "../modules/test"
+  prefix = "cq"
```

Comment:
The default prefix should be different than the prefix in the local one. If we have it the same then you can only deploy the terraform to a single region (unless you change the prefix)